static/templates/admin_user_list.handlebars

```diff
@@ -16,6 +16,8 @@
     {{else if is_active}}
     <td class="last_active">
     </td>
+    {{else}}
+    <td class="last_active"></td>
```

Comment:
I'm a bit skeptical of this change without a corresponding change in the `th` code.  Or was that broken before?

Also, don't you want to just remove `else if` above if this is correct?

Reply:
This was already broken before — we never updated the table in a condition where there would be an inactive user in the table, but that case would just yield a column short of what it would have otherwise.

Good catch on the `else` logic, I didn't catch that. Updated.